packages/react/cypress/support/commands.js

```diff
@@ -3,6 +3,5 @@ import compareSnapshotCommand from 'cypress-image-diff-js/dist/command';
 import '@testing-library/cypress/add-commands';
 import '@4tw/cypress-drag-drop';
 import 'cypress-file-upload';
-import 'cypress-plugin-tab';
```

Comment:
does this need to be removed for this PR?

Reply:
Kinda of yeah. I didn't know the realPress('tab') existed when I created it, and after I switched to "click" to maintain focus the existing keyboard navigation test quit working because the tab plugin is much flakier than the `realPress` function. After replacing `tab()` with `realPress('tab')` the tests passed every time instead of 1/4 times.